Celeste.Mod.mm/Mod/Everest/Everest.Loader.cs

```diff
@@ -294,12 +297,25 @@ public static class Loader {
                             }
                             continue;
                         }
+                        if (entry.FileName == ".everestignore") {
+                            List<string> lines = new();
```

Comment:
```suggestion
                            List<string> lines = new List<string>();
```
Everest code style only uses target-typed new for horrendously long types.